src/tools/auth0/handlers/emailTemplates.ts

```diff
@@ -23,7 +23,7 @@ export default class EmailTemplateHandler extends DefaultHandler {
     super({
       ...options,
       type: 'emailTemplates',
-      id: 'template',
+      identifiers: ['template'],
```

Comment:
The keyword preservation algorithm will be leveraging the `identifiers` field for all handlers. This is why the `id` field was removed in favor of `identifiers` here. Because email templates does not use the normal diff'ing algorithm as most other resource in the codebase, this will have no negative effects.